Makefile

```diff
@@ -125,6 +139,15 @@ toc-verify:
 vet: ## Run go vet against code.
 	$(GO_CMD) vet ./...
 
+## Create Kind Cluster for E2E
+.PHONY: kind-create
+kind-create:
```

Comment:
```suggestion
kind-create: kind
```

Reply:
Same for other directives that need the kind binary.

Reply:
typically I'd put creating, deleting, and any other kind related steps in a shell script

that script should have an EXIT trap that ensures the cluster is deleted and logs are dumped prior to deletion for debugging with `kind export logs $ARTIFACTS`, in case the test is terminated early. I don't think make has a good way to do this.

see: https://github.com/kubernetes-sigs/kind/blob/main/hack/ci/e2e-k8s.sh